css/index.css

```diff
@@ -1,3 +1,358 @@
+	padding: 10px; 
+    text-align: center;
+}
+
+h1 {
+    text-align: center;
+	font-size: 80px;
+	/* position: relative;  */
+	display: flex; 
+	/* padding: 70px 70px;
+	margin-bottom: -50px; */
+}
+
+
+button {
+	font-size: 20px;
+	padding: 5px 40px 5px 40px;
+    display: flex;  
+	background-color: white;
+    border: 1px solid black;
+    height: 40px;
+    margin-top: 55%;
+    margin-left: -50%;
+}
+
+
+h2 {
+    font-size: 25px;
+}
+
+.headerImage {
+	margin-left: 200px;
+	margin-top: 60px;
+	margin-bottom: 60px;
+
+}
+
+.topsection {
+	display: flex; 
+	width: 100%; 
+	vertical-align: middle;
+}
+
+.topsection-text {
+	display: flex; 
+	text-align: center; 
+	vertical-align: top;
+	margin-top: 110px;
+}
+	
+
+header {
+	margin-left: 100px;
+	vertical-align: middle; 
+}
+
+.getItCentered {
+    display: flex; 
+    
+}
+
+.middle-img {
+	width: 98%;
+	box-align: center; 
+	padding-top: 20px;
+
+}
+
+body {
+	width: 1200px;
+	margin: 0 auto;
+
+}
+
+.top {
+	width: 100%;
+}
+
+
+.features,
+.about {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 2%;
+	/* width: 49%;
+    padding: 30px; */
+}
+
+.services, 
+.product,
+.vision {
+    display: flex;
+    flex-wrap: wrap;
+    margin: 2%;
+	/* width: 33%;
+	padding: 30px; */
+	/* border-bottom: 1px solid black; */
+	
+}
+
+.contact {
+	margin-top: 30px;
+	margin-left: 10px;
+}
+
+.copyright {
+	text-align: center;
+}
+
+.top-content {
+    border-top: 2px solid black;
+    display: flex;
+    /* flex-wrap: wrap;
+    flex-flow: row nowrap;  */
+    flex-direction: row;
+    width: 100%
+
+}
+
+.bottom-content {
+    border-bottom: 2px solid black;
+    display: flex;
+    flex-flow: row nowrap;
+}
+
+/* responsive design starts here! */
+ 
+/* TABLET DISPLAY */
+@media screen and (max-width: 800px) {
+    body {
+        width: 100%;
+        margin: 0 auto;
+        overflow-x: hidden;
+    }
+
+    .headerImage {
+        display: none;
+    }
+
+    .topsection {
+        display: flex;
+        justify-content: center;
+        margin-bottom: 5%;
+    }
+
+    .topsection-text {
+        display: flex;
+        justify-content: center;
+        margin-top: 10%;
+    }
+
+    h1 {
+        margin-left: -30%; 
+    }
+
+    .top-content {
+        display: flex;
+        justify-content: center;
+    }
+
+    .logo {
+        position: absolute; 
+        padding-left: 38%;
+    }
+
+    nav a {
+        margin-top: 10%
+    }
+}
+
+    /* MOBILE DISPLAY */
+
+    @media screen and (max-width: 500px) {
+        body {
+            width: 100%;
+            margin: 0 auto;
+            overflow-x: hidden;
+        }
+
+        nav {
+            display: flex;
+            flex-direction: column;
+            /* margin-left: 50%; */
+            margin-top: 20%;
+        }
+
+        nav a {
+            display: flex;
+            flex-direction: column;
+            margin-left: 50%;
+            margin-top: 5%;
+            /* border-top: 1px solid black; */
```

Comment:
Right now, this border doesn't look good because it doesn't extend all the way right-to-left. If you want it to go all the way right-to-left, you can remove the `margin-left: 50%;` and add a `width: 100%;` property. The text for each a tag will stay centered, and the border will extend all they way. Afterwards, you can tweak it a little bit with the margin-top to make the a tags look even up-and-down.